src/shared/model/QueryParams.ts

```diff
@@ -20,7 +21,7 @@ export interface PersistableQueryParams
   extends TrackingQueryParams,
     StringifiableRecord {
   returnUrl: string;
-  clientId?: string;
+  clientId?: typeof validClientId[number];
```

Comment:
I know this is already merged, but could this type not just be `ValidClientId` (that is exported here https://github.com/guardian/gateway/pull/1273/files#diff-ae8ba4ba4c4c6df1c98f5fc76009bd75c0c658fc4ebff6b591d68a672dfa9c1fR9)

Reply:
ah yeah, I added this and forgot to change to use that. good spot! Another jobs PR is likely coming, so I can change it in that